Readme.md

```diff
@@ -44,6 +44,17 @@ class Post < ActiveRecord::Base
 end
 ```
 
+You can pass attributes to specify which enumeratior and which column to use:
+
+```ruby
+class Post < ActiveRecord::Base
+  enumeration :status,
+              foreign_key: :post_status_id, # specifies which column to use
+              class_name: 'Post::Status' # specifies the class of the enumerator
```

Comment:
Yes, because of this:

https://github.com/infinum/enumerations/blob/4a33b30b4a5044da5f50bbf217d27ce83b6b97d9/lib/enumerations.rb#L68

You can't call constantize on constant.
Maybe this would be best solution:

`options[:class_name].to_s.constantize.find(send(options[:foreign_key]))`